src/assertions/toExist/index.js

```diff
@@ -0,0 +1,31 @@
+import { AWSClient } from "../../helpers/general";
+
+export default {
+  async toExist(bucketName) {
```

Comment:
The name isn't the most obvious but it makes sense? expect("bucketName").toExist()
Any better suggestions?

Reply:
`toExistAsS3Bucket`?